src/main/java/frc/robot/commands/DefaultArmCommand.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.constants.ArmConstants;
+import frc.robot.constants.JoystickConstants;
+import frc.robot.subsystems.ArmControlSubsystem;
+import frc.robot.subsystems.Input;
+
+public class DefaultArmCommand extends CommandBase {
+  
+  private final ArmControlSubsystem mArmControlSubsystem;
+
+  public DefaultArmCommand(ArmControlSubsystem armControlSubsystem) {
+    mArmControlSubsystem = armControlSubsystem;
+    addRequirements(mArmControlSubsystem);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    handleInput();
+  }
+
+  void handleInput(){
+
+    if(Input.getLeftStickY() > JoystickConstants.PIVOT_INPUT_DEADZONE){
```

Comment:
Did you try saving all the files with the updated VS Code formatting settings that Mr Doan checked in? I ask cause the format here doesn't look quite right, spaces are missing in I think.